test/e2e/pkg/metadata/metadata.go

```diff
@@ -0,0 +1,7 @@
+package metadata
+
+type OperatorNamespace struct {
+	Name string
+}
+
+var Namespace = OperatorNamespace{}
```

Comment:
It's unused now

Reply:
Will be necessary in the future to collect data in PROW